go.mod

```diff
@@ -19,3 +19,84 @@ require (
 	k8s.io/client-go v0.23.6
 	sigs.k8s.io/controller-runtime v0.11.2
 )
+
+require (
```

Comment:
what's this about?

Reply:
The flip on line 3 from go 1.16 to 1.17 did that.